lib/web/apiserver_login_test.go

```diff
@@ -337,6 +346,158 @@ func TestAuthenticate_rateLimiting(t *testing.T) {
 	}
 }
 
+func TestAuthenticate_deviceWebToken(t *testing.T) {
+	pack := newWebPack(t, 1 /* numProxies */)
+	authServer := pack.server.AuthServer.AuthServer
+	proxy := pack.proxies[0]
+	clock := pack.clock
+
+	// Mimic a successfully create DeviceWebToken, regarldess of any parameters.
```

Comment:
```suggestion
	// Mimic a valid DeviceWebToken, regardless of any parameters.
```

Reply:
This godoc was all kinds of terrible, thanks for spotting it.